### PR TITLE
PLAT-385: Update coregroup endpoint

### PR DESCRIPTION
**Pull request by docktorrr**
## Purpose
Change core group endpoint JSON according to: https://humanitec.atlassian.net/browse/PLAT-385

## Approach
- Created custom PermissionField
- Added m2m workflows fields in the serializer
- Remove old viewset for django built-in permissions

### CODE REVIEW

File: workflow/serializers.py [hunk 1]
```diff
@@ -17,10 +16,28 @@
...
+    """
+    Field for representing permissions as a JSON opbject
+    """
+    _keys = ('create', 'read', 'update', 'delete')
+
+    def __init__(self, *args, **kwargs):
+        kwargs['child'] = serializers.BooleanField()
+        super().__init__(*args, **kwargs)
+
+    def to_representation(self, value):
```
Comment (ralfzen) [inline workflow/serializers.py:31]: It would be nice to have some documentation here.

**Review by ralfzen — CHANGES_REQUESTED** [review]
All in all, looks good, but it's hard to understand. Some explaining docs strings or comments would be nice. Thank you.

File: workflow/serializers.py (hunk 1 above)
Comment (docktorrr) [reply workflow/serializers.py:31]: But it's quite low-level code, I don't think that we need to explain every line of code in this case :)

For PermissionField doc string - yes, I'll extend the description.

**Review by ralfzen — APPROVED** [review]
awesome. Thanks!

**Pull request merged**